### PR TITLE
Create 2023-09-22-OpenSearch-Personas-Creating-Figma-Templates-to-Rep…

### PR DESCRIPTION
**Pull request by apasun**
…resent-Person-Framework

This blog gives community members the tools to create personas an jobs to be done. It explains how to access and use these tools.

### Description
New Blog regarding the access and use of Figma Templates to create personas and jobs to be done
 

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by pajuric** [comment]
@nateynateynate @krisfreedain - This blog is ready to publish today.

File: _posts/2023-09-22-OpenSearch-Personas-Creating-Figma-Templates-to-Represent-Person-Framework [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "OpenSearch search personas: Creating Figma templates to represent a persona framework"
+authors: 
+  - apasun, macrakis
```
Comment (krisfreedain) [inline _posts/2023-09-22-OpenSearch-Personas-Creating-Figma-Templates-to-Represent-Person-Framework:5]: should be two lines 

```
authors: 
  - apasun
  - macrakis
```

File: _posts/2023-09-22-OpenSearch-Personas-Creating-Figma-Templates-to-Represent-Person-Framework [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+date: 2023-09-22
+meta_keywords: OpenSearch persona, jobs to be done, figma templates
+meta_description: Read on to learn how to use the journey map library on Figma
+twittercard:
+  description: Learn about our our active community channel on Figma and how to use persona and jobs to be done templates
+  - community
+excerpt: 
+
```
Comment (krisfreedain) [inline _posts/2023-09-22-OpenSearch-Personas-Creating-Figma-Templates-to-Represent-Person-Framework:13]: missing end of meta section marker

`---`

File: _posts/2023-09-22-OpenSearch-Personas-Creating-Figma-Templates-to-Represent-Person-Framework [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+#### **OpenSearch search personas: Creating Figma templates to represent a persona framework**
```
Comment (krisfreedain) [inline _posts/2023-09-22-OpenSearch-Personas-Creating-Figma-Templates-to-Represent-Person-Framework:15]: let's drop this extra line, it ends up throwing off the main blog page

File: _posts/2023-09-22-OpenSearch-Personas-Creating-Figma-Templates-to-Represent-Person-Framework [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+The OpenSearch Project has developed a Figma library for the community. The library offers resources to help you create persona decks, customer journey maps, and flow diagrams that can be used to present user research to your team. This blog post provides information on how to use this library. The welcome page gives you information about the library and how to use it. You can use this component library as a local file or share it with your team. Other provided information includes how to select and use components and component properties and how to customize components for your use. 
+[Image: OpenSearch Journey Map Library.jpg]
```
Comment (krisfreedain) [inline _posts/2023-09-22-OpenSearch-Personas-Creating-Figma-Templates-to-Represent-Person-Framework:22]: can you include this image?

File: _posts/2023-09-22-OpenSearch-Personas-Creating-Figma-Templates-to-Represent-Person-Framework [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+[Image: image]
+The sticker sheet provides an overview of the components as well as some variants. In addition, we provided a sample journey map so that you can obtain an idea of how to lay out your own journey map in accordance with your team’s requirements. 
+[Image: Sticker sheet 1.jpg][Image: image.png]Here is an example of a journey map template and one variation (dark mode).
```
Comment (krisfreedain) [inline _posts/2023-09-22-OpenSearch-Personas-Creating-Figma-Templates-to-Represent-Person-Framework:25]: I don't see the image uploaded at part of this PR

File: _posts/2023-09-22-OpenSearch-Personas-Creating-Figma-Templates-to-Represent-Person-Framework [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+[Image: image.png]
+Now you can create journey maps incrementally based on the persona. The Figma template offers various components corresponding to the user’s phase, stage, what they aim to do, how they feel, how they think, the tools they use, pain points, opportunities, the level of effort needed to accomplish the phase, and the stakeholders involved at that point in time. The template is designed so that you can build the journey map step by step. You can duplicate each phase or create a specific component individually. Ultimately, once you have constructed each user phase, you can align various components as appropriate. Now you can select a frame and choose different themes (such as dark mode or color templates that you can predefine). The OpenSearch Project is in the process of creating core personas for search, observability, and security use cases. We will be using these templates to define users and journeys, and we hope you will find this useful in defining your users and experiences. 
+
+More OpenSearch Project resources available on Figma: https://www.figma.com/@opensearch
```
Comment (krisfreedain) [inline _posts/2023-09-22-OpenSearch-Personas-Creating-Figma-Templates-to-Represent-Person-Framework:32]: add `<>` around all the links to make them clickable -- example:

`<https://www.figma.com/@opensearch>`

**Review by krisfreedain — CHANGES_REQUESTED** [review]
Hello @apasun @pajuric -- a few edits needed

File: _posts/2023-09-22-OpenSearch-Personas-Creating-Figma-Templates-to-Represent-Person-Framework (hunk 1 above)
Comment (apasun) [reply _posts/2023-09-22-OpenSearch-Personas-Creating-Figma-Templates-to-Represent-Person-Framework]: Edited

File: _posts/2023-09-22-OpenSearch-Personas-Creating-Figma-Templates-to-Represent-Person-Framework (hunk 4 above)
Comment (apasun) [reply _posts/2023-09-22-OpenSearch-Personas-Creating-Figma-Templates-to-Represent-Person-Framework:23]: Yes, I will add images

**Review by krisfreedain — APPROVED** [review]
OK - fixed everything - looks like we're good now @apasun

**Pull request merged**